modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/view_results_rankings.jsp

```diff
@@ -136,24 +132,102 @@ RankingPortletDisplayContext rankingPortletDisplayContext = (RankingPortletDispl
 	</liferay-ui:search-container>
 </aui:form>
 
-<aui:script>
+<aui:script sandbox="<%= true %>">
+	var activateResultsRankingsEntries = function() {
+		var searchContainer = document.getElementById(
+			'<portlet:namespace />resultsRankingEntries'
+		);
+
+		if (searchContainer) {
+			Liferay.Util.postForm(
+				document.<portlet:namespace />resultsRankingEntriesFm,
+				{
+					data: {
+						activateFormInstanceIds: Liferay.Util.listCheckedExcept(
+							searchContainer,
+							'<portlet:namespace />allRowIds'
+						)
+					},
+
+					<portlet:actionURL name="/results_ranking/edit" var="activateResultsRankingEntryURL">
+						<portlet:param name="<%= Constants.CMD %>" value="<%= ResultRankingsConstants.ACTIVATE %>" />
+						<portlet:param name="redirect" value="<%= currentURL %>" />
+					</portlet:actionURL>
+
+					url: '<%= activateResultsRankingEntryURL %>'
+				}
+			);
+		}
+	};
+
+	var deactivateResultsRankingsEntries = function() {
+		if (
+			confirm(
+				'<liferay-ui:message key="are-you-sure-you-want-to-deactivate-this" />'
+			)
+		) {
+			var searchContainer = document.getElementById(
+				'<portlet:namespace />resultsRankingEntries'
+			);
+
+			if (searchContainer) {
+				Liferay.Util.postForm(
+					document.<portlet:namespace />resultsRankingEntriesFm,
+					{
+						data: {
+							deactivateFormInstanceIds: Liferay.Util.listCheckedExcept(
+								searchContainer,
+								'<portlet:namespace />allRowIds'
+							)
+						},
+
+						<portlet:actionURL name="/results_ranking/edit" var="deactivateResultsRankingEntryURL">
+							<portlet:param name="<%= Constants.CMD %>" value="<%= ResultRankingsConstants.DEACTIVATE %>" />
+							<portlet:param name="redirect" value="<%= currentURL %>" />
+						</portlet:actionURL>
+
+						url: '<%= deactivateResultsRankingEntryURL %>'
+					}
+				);
+			}
```

Comment:
Do you think we can reduce the repetitiveness of this?

Since it seems like the only difference is the `url`, maybe we can have a `submitForm` function so then this section could look something like:
```jsp
<portlet:actionURL name="/results_ranking/edit" var="deactivateResultsRankingEntryURL">
	<portlet:param name="<%= Constants.CMD %>" value="<%= ResultRankingsConstants.DEACTIVATE %>" />
	<portlet:param name="redirect" value="<%= currentURL %>" />
</portlet:actionURL>

submitForm('<%= deactivateResultsRankingEntryURL %>');
```